CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+# Contribute to the documentation
+
+Thank you for your interest in contributing to the MetaMask developer documentation!
+
+## Table of contents
+
+- [Contribution workflow](#contribution-workflow)
+  - [Preview locally](#preview-locally)
+- [Style guide](#style-guide)
+  - [Images](#images)
+- [Markdown guide](#markdown-guide)
+  - [Simplified tabs](#simplified-tabs)
+  - [Live code blocks](#live-code-blocks)
+
+## Contribution workflow
+
+The MetaMask documentation contribution workflow involves proposing changes by creating
+[branches](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches)
+and
+[pull requests (PRs)](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+on this repository.
+This facilitates open contributions, testing, and peer review.
+
+To contribute changes:
+
+1. Search for an [existing issue](https://github.com/MetaMask/metamask-docs/issues) to work on, or
+   [create a new issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue)
+   describing the content issue you'd like to address.
+   Make sure no one else is assigned to the issue, and assign yourself to it.
+   If you don't have permission to assign yourself to it, leave a comment on the issue.
+
+2. [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+   this repository to your computer.
+
+    ```bash
+    git clone https://github.com/MetaMask/metamask-docs.git
+    ```
+
+3. [Create and checkout a topic branch](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging),
+   naming it appropriately.
+   We recommend including the issue number and a short description in the branch name (for example,
+   `183-doc-cli-option`), which is a reminder to fix only one issue in a PR.
+
+    ```bash
+    git checkout -b <ISSUE-NUM>-<ISSUE-DESC>
+    ```
+
+   > **Tip:** You can use a Git client such as [Fork](https://fork.dev/) instead of the command line.
+
+4. Open this repository in a text editor of your choice (for example,
+   [VS Code](https://code.visualstudio.com/)) and make your changes.
+   Refer to the [style guide](#style-guide) and [Markdown guide](#markdown-guide) when making
+   documentation changes.
+
+   > **Notes:**
+   > - All documentation content is located in the `wallet` and `snaps` directories.
+   > - If you add a new documentation page, make sure to edit `wallet-sidebar.js` or
+       `snaps-sidebar.js` to [add the page to the sidebar](https://docusaurus.io/docs/sidebar/items).
+   > - If you delete or rename a documentation file, make sure to add a redirect to the
```

Comment:
rename probably means move as well, but perhaps we should make this, "delete, rename, or move..."?